rdt-open-reader/src/main/java/io/ona/rdt/activity/PatientRegisterActivity.java

```diff
@@ -166,7 +179,7 @@ public void onSyncInProgress(FetchStatus fetchStatus) {
 
     @Override
     public void onSyncComplete(FetchStatus fetchStatus) {
-        // TODO: implement this
+        updateSyncDate();
```

Comment:
We need to check that the fetchStatus is a success.

Reply:
done